database.sql

```diff
@@ -0,0 +1,36 @@
+CREATE DATABASE family;
+
+USE family;
+
+CREATE TABLE `users` (
+	`id` INT(10) NOT NULL AUTO_INCREMENT ,
+	`name` VARCHAR(25) NOT NULL,
```

Comment:
what if fullname (name, lastname, middlename) consist of more than 25 symbols?
in this camse mysql will cut all rest symbols
i think you can use here even more then 25
i'd suggest you to google how many symbols developers leave for a full name field